### ISSUE TITLE
Missing Akismet causes fatal error on Jetpack Dashboard

### ISSUE THREAD

**Issue by richardmtl** [Automattic/jetpack#8399]
Reported in: 817571-zen

Note: This is the second time I encounter this. I do not remember if I reported it the last time, nor can I find it anywhere, so, maybe I didn't report it, sorry. ☹️ It's odd that it's not coming up more often though.

Here is the relevant bit from the error log, generated after visiting the Jetpack page in the dashboard (site has Jetpack 5.6.1):

```
[20-Dec-2017 19:46:03 UTC] PHP Fatal error:  require_once(): Failed opening required '/home/content/[REDACTED]/html/wp-content/plugins/akismet/class.akismet.php' (include_path='.:/usr/local/php5_6/lib/php') in /home/content/[REDACTED]/html/wp-content/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php on line 422
```
I've suggested that the user just install Akismet for now. Any ideas why this is happening in the first place, though? Shouldn't we not care if Akismet is there or not? Maybe there's something more going on..?

**Comment by oskosk** [comment]
@richardmtl the two lines before line `422` mentioned there specifically check that the akismet files are there... So it looks like a permissions issue. 

With my browser I tried to load the following paths on that site and got these responses:

*  `wp-content/plugins/jetpack/readme.txt` 200 - OK
*  `wp-content/plugins/jetpack/asdasd.txt` 404 - Not found
*  `wp-content/plugins/akismet/readme.txt` 403 - Forbidden
*  `wp-content/plugins/akismet/asdasd.txt` 403 - Forbidden

So I would imagine that there's a permissions issue at least on the `plugins/akismet` dir if it's there.

**Comment by richardmtl** [comment]
@oskosk Akismet wasn't installed at the time, at least on the Plugins page; maybe the folder was there, a remnant of a previous install or update that failed? As I mentioned though, this wasn't the first time I saw this particular issue. What do you think about that possibility?

**Comment by oskosk** [comment]
If the permissions are wrong (`+x` for the `akismet` dir and `-r` for the files inside) it's probable that the plugins page won't show the plugins either. WordPress can't read the plugin info on the files.

**Comment by oskosk** [comment]
For me it looks like the folder AND the files are inside...

**Comment by oskosk** [comment]
We could do some extra check for these cases where the plugin seems to be there but unreadable by Jetpack... Would this help anyways ?

**Comment by oskosk** [comment]
This edge case is addressed by #8402

**Comment by oskosk** [comment]
@richardmtl  The fix for making Jetpack not crash in this situation has been merged. Will be shipping with Jetpack 5.7. Left a note on 817571-zen because the user will still have problems with Akismet.
